common/sgx/tcbinfo.c

```diff
@@ -399,9 +402,154 @@ static oe_result_t _read_tcb_info_tcb_level(
     return result;
 }
 
+/**
+ * Parse the tcbcomponent object
+ * at JSON object.tcbInfo.tcbLevels[].tcb.<sgx|tdx>tcbcomponents[]
+ *
+ * Only return the svn as other fields are not used
+ *
+ * tcbcomponent Schema
+ * {
+ *     "svn": integer,
+ *     "category": string (optional),
+ *     "type": string (optional)
+ * }
+ */
+static oe_result_t _read_tcb_info_tcb_level_tcb_tcbcomponent(
```

Comment:
```suggestion
static oe_result_t _read_tcb_info_tcb_level_tcb_tcbcomponent_v3(
```

If this is specific to the v3 format.

Reply:
Yes